group/templates/group/editAnnounce.html

```diff
@@ -0,0 +1,60 @@
+{% extends "index/base.html" %}
+{% load static %}
+{% block title_name %}
+<title>Edit for Announce...</title>
+{% endblock title_name %}
+{% load bootstrap %}
+{% block body_block %}
+<br>
+<br>
+<div class="col-lg-offset-1 col-lg-10">
+  <form action="." method="POST">
+    {% csrf_token %}
+    <div class="alert alert-dismissable alert-info">
+      <button type="button" class="close" data-dismiss="alert">×</button>
+      <h4>Hint: <span style="color:red">Title</span> is required!</h4>
+    </div>
+    {% if user_is_owner or user_is_coowner or user_has_admin_auth %}
+      <a href="" data-toggle="modal" data-target="#deleteGroupComfirm">
+        <button type="button" class="btn btn-danger btn-xs" style="float:right">
+          Delete this announce
+        </button>
+      </a>
+      <!--Modal for Delete_Group_Comfirm-->
+      <div class="modal fade" id="deleteGroupComfirm" role="dialog" aria-hidden="true">
```

Comment:
oh, I have no idea about that.
@bruce3557 @henryyang42 do you have any suggestion?